bootstrap.sh

```diff
@@ -2,7 +2,9 @@
 
 apt-get update
 apt-get install python3 python3-pip -y
-pip3 install -r /vagrant/requirments.txt
+sudo pip3 install Flask Flask-SQLAlchemy flask_login flask_wtf PyMySQL
```

Comment:
Why doesn't it work? Is it in the correct directory?

Reply:
yah i think it is.
i think its some wierd bug i think

Reply:
if you can merge this PR it would be great, for now we will work with this- it is important so that i can try to get the ci to work. thanks!